### ISSUE TITLE
Status: OWL/OWC Page incorrectly formatted/error

### ISSUE THREAD

**Issue by Tokovg** [shirokumacode/overwatch-omnic-rewards#23]
On the site the broadcast worked, the indicator too.

I waited more than 10 minutes when the check every minute is set in the settings. After clicking "check now" the error disappeared. But in a copy of the program for another account, it was still displayed until I checked it manually.

**After this error, the program stops earning rewards and checking the broadcast until you manually restart or click "check now".**

I suspect that it was because of this error that I received only 10 tokens instead of more than 50 for yesterday.

**Comment by shirokumacode** [comment]
That error has been happening a lot in this season. I haven't seen it at all last season. This means that overwatch league website has no video panel and/or is displaying an error. It seems this is very random and if you go to overwatchleague.com exactly when this happens you should see it clearly. 
Naively when I programmed this app I assumed this error is kinda catastrophic and would never happened (or at least would happen only when Blizzard would change the format of the website aka app breaking)  and so this error makes the app stop checking overwatchleague.com forcing you to perform a check now/restart to start working again. This was to prevent useless requests to overwatchleague.com. And so if this error happens the app will need manually intervention unfortunately.
I will update the app with better error handling making manual intervention unnecessary. If any error like this, the app simply will return to a checking mode aka will check every x min if the stream is live.

**Comment by shirokumacode** [comment]
I released a new version with better error handling (v1.3.2) that handles this better. Now it doesn't require to manually restart or click "check now" as it goes back to a checking mode automatically. I also reduced the default checking time from 10min to 5min to avoid possible time not tracked while Blizzard's tracking endpoints or the website are being finicky. 

Closing this for now. Will reopen if these mitigations are not enough. Hopefully Blizzard fixes their stuff.
